src/python/dxpy/nextflow/nextflow_builder.py

```diff
@@ -32,15 +32,18 @@ def build_pipeline_from_repository(args):
             "Can't create an applet without specifying a destination project; please use the -d/--destination flag to explicitly specify a project")
     input_hash = {
         "repository_url": args.repository,
+        "repository_tag": args.tag,
```

Comment:
Which method the docstring from line 23 applies to? Should it perhaps be more informative, eg (?):

Creates files needed to build a nextflow applet and returns a folder with these files (dxapp.json, src/code.sh, etc). The folder also includes resources/ which contains the nextflow pipeline. Note that folder is created as a tempfile.

Please add a short description to this module itself as well. Thanks!

Reply:
Also, per our discussion, please add docstrings. One example you can follow is [here](https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/bindings/dxfile.py#L651).